### DISCUSSION TITLE
v0.2.1

### DISCUSSION THREAD

**Discussion by CarsCupcake** [CarsCupcake/SkyblockRemake#17] [General]
### Private Isle Update
**This Version is all about adding some private isle features.**

List of added Features:
- Private Isle island type (you have to set the Server Type to "private" to test it out)
- Cobblestone minion
- Farming Tools
- Update Checker
- A proper skill menu where you can finaly see your skill progress/level

<hr /><em>This discussion was created from the release <a href='https://github.com/CarsCupcake/SkyblockRemake/releases/tag/0.2.1'>v0.2.1</a>.</em>

**Comment by TheGod5** [comment]
Add skyblock menu (Nether star)

**Reply by CarsCupcake** [reply]
is added ;)

**Comment by space713** [comment]
what version is the plugin 
also it looks cool

**Reply by CarsCupcake** [reply]
1.17.1 ( as you can see in the readme ;) )
